conf/network/ganache-config.js

```diff
@@ -10,11 +14,21 @@ const CONTRACT_DEFINITIONS = {
   TokenFRTProxy: CONTRACTS_BASE_DIR + '/TokenFRTProxy',
   TokenOWL: CONTRACTS_BASE_DIR + '/TokenOWL',
   TokenOWLProxy: CONTRACTS_BASE_DIR + '/TokenOWLProxy',
-  TokenGNO: CONTRACTS_BASE_DIR + '/TokenGNO'
+  TokenGNO: CONTRACTS_BASE_DIR + '/TokenGNO',
+  Safe: CONTRACTS_SAFE_MODULE_DIR + '/GnosisSafe',
+  SafeDXCompleteModule: CONTRACTS_SAFE_MODULE_DIR + '/DutchXCompleteModule',
+  SafeDXSellerModule: CONTRACTS_SAFE_MODULE_DIR + '/DutchXSellerModule'
+}
+
+const SAFE_MODULE_ADDRESSES = {
```

Comment:
So the address are specified? I mean, cannot be read from the NPM package so it works for any network? 
Or it is because u need to always set the safe address, because every bot need specify the addresses because it's deployed ad hoc per project.